### ISSUE TITLE
Improve Accessibility Testing

### ISSUE THREAD

**Issue by jasonfu9966** [ustaxcourt/ef-cms#8714]
## Acceptance Criteria
Accessibility tests are no longer flaky / run consistently

## Pain Avoided/Frustration Saved
- We won't have to re-run flaky tests when opening PRs

## Breadth/Pervasiveness of Problem
  - medium/high

## Complexity of Problem (Low, Medium, High) and Why it's Complex
  - medium

## Notes
  - a consistently-flaky test is in `test:pa11y:2`, most likely in the `irsPractitioner` section
    - this test failure is reproduce-able locally, but only about 5% of the time
  - keeping pa11y as a library is not a requirement of this story

**Comment by rachelschneiderman** [comment]
@ttlenard @cholly75  
Things we changed that need regression testing:
- Section document Qc checkboxes
- All modals have been updated, users can no longer click "outside" of modals to close them. They must make a selection on the modal to close it.
- Calendaring trial session, eligible cases checkboxes
- Pending report page header
- Edit orders needed on new petition checkboxes
- Case detail menu buttons
- Before starting a case, "Next you’ll continue to follow the steps to upload..." text
- Accessibility statement

**Comment by ttlenard** [comment]
- [x] Section document Qc checkboxes
- [x] All modals have been updated, users can no longer click "outside" of modals to close them. They must make a selection on the modal to close it.
     - [x] Docket Clerk - Service in Consolidated Case to select all or some cases
     - [x] Docket Clerk - Ready to send Paper Service?
     - [x] Docket Clerk - Add Consolidated Case
     - [x] Docket Clerk - Remove Consolidated Case
     - [x] Docket Clerk - Set a case for hearing
     - [x] Docket Clerk - Add/Edit Calendar Note
     - [x] Docket Clerk - Remove case from Trial
     - [x] Docket Clerk - Edit Case Status/Caption
     - [x] Docket Clerk - Seal/Unseal a case
     - [x] Docket Clerk - Strike Entry
     - [x] Docket Clerk - Printable Docket Record
     - [x] Docket Clerk - Download Docket Entries
     - [x] Docket Clerk - Seal/Unseal Document 
     - [x] Docket Clerk - Remove Pending item 
     - [x] Docket Clerk - Add/Edit/Delete Deadline
     - [x] Docket Clerk - Create Order/Notice
     - [x] Docket Clerk - Edit Order Title
     - [x] Docket Clerk - Add/Edit Docket numbers to a Consolidated Order
     - [x] Docket Clerk - Remove signature from draft order
     - [x] Docket Clerk - Delete Draft Document 
     - [x] Docket Clerk - Add Petitioner Counsel
     - [x] Docket Clerk - Seal Petitioner Address
     - [x] Docket Clerk - Remove Counsel
     - [x] Docket Clerk - Cancel out of pages
     - [x] Docket Clerk - Calculate Penalties on IRS Notice
     - [x] Docket Clerk - Calculate Penalties as Determined by Court
     - [x] Docket Clerk - Itemized Penalties
     - [x] Docket Clerk - Create/Complete/Forward/Reply/Complete and Send Message
     - [x] Docket Clerk - Add/Edit Case Notes
     - [x] Docket Clerk - Run Trial Session Planning Report
     - [x] Docket Clerk - Run Case Inventory Report
     - [x] Docket Clerk - Print Paper Service PDF
     - [x] Petitioner - DAWSON Update
- [x] Calendaring trial session, eligible cases checkboxes
- [x] Pending report page header
- [x] Edit orders needed on new petition checkboxes
- [x] Case detail menu buttons
- [x] Before starting a case, "Next you’ll continue to follow the steps to upload..." text
- [x] Accessibility statement
